tools/gmhazard_scripts/db_creation/empirical_db/calc_distances_flt.py

```diff
@@ -215,7 +215,7 @@ def main():
         fault_df = pd.DataFrame(sorted(nhm_data.keys()), columns=["fault_name"])
 
         site_source_distance_data, directivity_data = compute_site_source_distances(
-            stations.to_numpy(), nhm_data, calculate_directivity=args.directivity
+            stations.to_numpy(), nhm_data, calculate_directivity=not args.no_directivity
```

Comment:
Instead of flipping the logic, for negative options I use dest to flip the logic, so we have positive boolean variables (I think positively named variables are more intuitive)

![image](https://user-images.githubusercontent.com/25143301/150012236-3c967fb1-dbdb-47a7-9c96-b30325297d33.png)